payment/app.py

```diff
@@ -1,9 +1,9 @@
 import os
 import atexit
 
-from flask import Flask
+from flask import Flask, Response
 import redis
-
+import Request
 
```

Comment:
The file is missing several imports: ```json```, ```random```. And I think you need ```requests``` instead of Request. ```requests``` should also be installed in the dependencies of the microservice.